supervisor/usb.h

```diff
@@ -58,10 +60,19 @@ typedef struct {
     size_t num_out_endpoints;
 } descriptor_counts_t;
 
+typedef struct {
+    uint16_t vid;
+    uint16_t pid;
+    char manufacturer_name[126];
+    char product_name[126];
```

Comment:
I don't understand this - if I put in `127` which is (as I understand) the correct value here, circuitpython crashes before setting up USB communications again. Anything less than 127 will work though apparently... But I don't see where this limit could come from?